types/qr-image/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for qr-image 3.2
+// Project: https://github.com/alexeyten/qr-image
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+/**
+ * error correction level. One of L, M, Q, H. Default M.
+ *
+ * @export
+ */
+export type ec_level = 'L' | 'M' | 'Q' | 'H';
+
+/**
+ * image type. Possible values png (default), svg, pdf and eps.
+ *
+ * @export
+ */
+export type image_type = 'png' | 'svg' | 'pdf' | 'eps';
+
+/**
+ * image options object:
+ *
+ * @export
+ * @interface IOptions
+ */
+export interface Options {
+	ec_level?: ec_level;	// error correction level. One of L, M, Q, H. Default M.
+	type?: image_type;		// image type. Possible values png(default), svg, pdf and eps.
+	size?: number;		// (png and svg only) for png and undefined for svg.-(png and svg only) — size of one module in pixels.
+	margin?: number;		// (only png)for png and 1 for others.-white space around QR image in modules.
+	parse_url?: boolean;	// (experimental, default false) try to optimize QR-code for URLs.
+}
+
+export function image(text: string, level_or_optoins?: ec_level | Options): NodeJS.ReadableStream;
```

Comment:
Just pointing out a typo. `optoins` -> `options`. Although now that you mention it, camelCase is typical for JS code.